com.github.miguelmota.Cointop.json

```diff
@@ -0,0 +1,60 @@
+{
+  "app-id": "org.flatpak.Hello",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.golang"
+  ],
+  "command": "cointop",
+  "modules": [
+    {
+      "name": "scripts",
+      "sources": [
+        {
+          "type": "script",
+          "commands": [
+            ". /usr/lib/sdk/golang/enable.sh;  export GOPATH=/app/go"
+            ],
+            "dest-filename": "my_enable.sh"
+        },
+        {
+          "type": "script",
+          "commands": [
+            ". /app/bin/scripts/my_enable.sh; cd /app/go/src/$1; go build -o x"
+            ],
+            "dest-filename": "build.sh"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/go/{src,pkg,bin} # make go workspace",
+        "install -d /app/bin/scripts",
+        "cp *.sh /app/bin/scripts/"
+      ]
+    },
+    {
+      "name": "hello",
+      "buildsystem": "simple",
+      "build-commands": [
+        "cp -rpv go/* /app/go/",
+        "/app/bin/scripts/build.sh github.com/miguelmota/cointop",
+        "ls /app/go/src/github.com/miguelmota/cointop",
+        "install -D /app/go/src/github.com/miguelmota/cointop/x /app/bin/cointop"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/miguelmota/cointop.git",
+          "commit": "70748c8e665e2608cdf88af91a180bcfd97dfcb1",
+          "dest": "go/src/github.com/miguelmota/cointop"
+        }
+      ]
+    }
+
+  ],
+  "finish-args": [
+    "--filesystem=host",
```

Comment:
I don't think you need file system access at all. Again once you hopefully fix the config path issue discussed.

Reply:
removed filesystem arg